Sources/MongoSwift/BSON/BSONEncoder.swift

```diff
@@ -38,14 +38,15 @@ public class BSONEncoder {
 
         let encoder = _BSONEncoder(options: self.options)
 
-        guard let topLevel = try encoder.box(value) else {
+        let boxedValue = try encoder.box_(value)
```

Comment:
nit: just do `guard let boxedValue = try ...`

Reply:
I always forget about `guard let` it seems. Done.